addons/website_event_sale/controllers/main.py

```diff
@@ -50,7 +50,7 @@ def registration_confirm(self, event, **post):
             order.action_confirm()  # tde notsure: email sending ?
             attendees = request.registry['event.registration'].browse(cr, uid, list(attendee_ids), context=context)
             # clean context and session, then redirect to the confirmation page
-            request.website.sale_reset(context=context)
+            request.website.sale_reset()
```

Comment:
Because we have removed old API arguments from original method(cr uid ids context). So if we pass those arguments here, it will give traceback for **unexpected keyword argument 'context'.**